### DISCUSSION TITLE
[@types/d3-geo] Missing `geoPath.digits` (introduced in v3.1.0)

### DISCUSSION THREAD

**Discussion by kachkaev** [DefinitelyTyped/DefinitelyTyped#67363] [Issues with a @types package] [answered]
At the time of writing, `@types/d3-geo@3.0.8` does not contain `geoPath.digits(n)` and `geoPath.digits()`. This method with two overloads was introduced in `d3-geo@3.1.0`. See [release notes](https://github.com/d3/d3-geo/releases/tag/v3.1.0), [docs](https://d3js.org/d3-geo/path#path_digits) and https://github.com/d3/d3-geo/pull/272.

I am opening this issue/discussion because I am working on https://github.com/airbnb/visx/pull/1767. In the meantime, I use `// @ts-expect-error` for the new method.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #67384 😊
